server/apps/main/views.py

```diff
@@ -458,6 +467,14 @@ def my_stories(request):
 
         # For each category, filter stories, create pagination and add continuous numbering
 
+        # All stories
+        paginator_all = Paginator(
+            files, items_per_page,
+        )
+        all_stories = paginate_stories(request, paginator_all, "page_all")
+        # Numbering for all stories different to all other tabs 
+        all_stories_numbering = list(range(all_stories.start_index(), all_stories.start_index()+min(items_per_page, len(all_stories.object_list))))
```

Comment:
This already works well as it is, but I'm wondering what the reason is that you can't pass `all_stories` into `number_stories()` like the others here. If you could it looks like it might simplify the `story_table.html` template.

Reply:
From my playing with things - you can only number a story once. So each story already has a number, and if you have one of each type you get 4 number 1s in the list. Or the "All Stories" numbering would overwrite the individual tab numbering. But this isn't something I really researched

Reply:
Oh, that's interesting. I guess the story dictionaries contain references to rather than copies of the stories; not necessarily a bad thing.

Reply:
In that case I can see it would take some refactoring to work around this; better to leave it as it is.